spacecmd/src/spacecmd/utils.py

```diff
@@ -614,6 +614,12 @@ def parse_str(s, type_to=None):
         if re.match(r'[1-9]\d*', s):
             return int(s)
 
+        if s == "True":
+            return True
+
+        if s == "False":
+            return False
+
```

Comment:
Just a suggestion to make it shorter:
```suggestion
        if s in ("False", "True"):
            return eval(s)

```